server/resources/aspects/rules/android/android_info.bzl.template

```diff
@@ -106,17 +107,17 @@ def extract_android_aar_import_info(target, ctx, dep_targets, **kwargs):
     if ctx.rule.kind != "aar_import":
         return None, None
 
-    if AndroidManifestInfo not in target:
+    if AndroidIdeInfo not in target:
         return None, None
-    manifest = file_location(target[AndroidManifestInfo].manifest)
+    manifest = file_location(target[AndroidIdeInfo].manifest)
 
     resource_folder = None
     r_txt = None
     if AndroidResourcesInfo in target:
-        direct_android_resources = target[AndroidResourcesInfo].direct_android_resources.to_list()
+        direct_android_resources = target[AndroidResourcesInfo].direct_resources_nodes.to_list()
```

Comment:
Could you do an if-condition, e.g. `if hasattr(target[AndroidResourcesInfo], "direct_resources_nodes"):` <new code> `else:` <old code>  so that both old and new versions of `rules_android` are supported?